model/api/src/main/java/org/jboss/aerogear/unifiedpush/dao/VariantMetricInformationDao.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * JBoss, Home of Professional Open Source
+ * Copyright Red Hat, Inc., and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.aerogear.unifiedpush.dao;
+import org.jboss.aerogear.unifiedpush.api.VariantMetricInformation;
+
+
```

Comment:
none of our DAO interface have javadoc on class level but I can add for this one. This interface is needed since we do queries on the VariantMetricInformation and DAO layer was missing for this entity